content/docs/howto/java.md

```diff
@@ -178,6 +178,56 @@ pack build samples/java \
 
 **NOTE:** It is important to properly quote values for `BP_INCLUDE_FILES` and `BP_EXCLUDE_FILES` as they may contain wild card characters like `*` which the shell may interpret. Proper quoting ensures the actual characters are passed through to the buildpack.
 
+
+##### Enable Maven or Gradle to build javascript assets with Node/Yarn 
+
+Build tools such as Maven & Gradle can configure plugins & tasks that build javascript assets for the frontend part of an app. The Java Buildpack can now install the `node` and `yarn` binaries for build tools to use in such tasks. 
+
+When building from source with the Maven or Gradle buildpacks, you can enable installation of Node and/or Yarn using the following environment variable:
+
+* `BP_JAVA_INSTALL_NODE` - set to `true` to enable, defaults to `false`.
+
+When this is set to `true`, the buildpack will check for the following files:
+
+* `yarn.lock` - both Yarn and Node will be installed via the [Yarn Buildpack][bp/yarn] & [Node Engine Buildpack][bp/node-engine]
+* `package.json` - only Node will be installed via the [Node Engine Buildpack][bp/node-engine]
+
+By default, the above files are expected to be in the application root directory. You can configure a specific sub-directory for these files using the environment variable supported by the node-engine buildpack:
+
+* `BP_NODE_PROJECT_PATH` - see [nodejs - Build an App from Source in a Subdirectory][nodejs-from-source]
+
+Example: Building a Java app & installing Yarn dependencies
+
+Sample Maven Plugin configuration (pom.xml):
+
+```plain
+<plugin>
+<groupId>org.codehaus.mojo</groupId>
+<artifactId>exec-maven-plugin</artifactId>
+<version>${exec-maven.version}</version>
```

Comment:
locally my maven did not resolve this property; used in a Spring Boot project pom, you don't need this version anyway, the BOM has it
I suggest you remove this line

Reply:
I didn't give a version here just to prevent the docs getting stale